### PR TITLE
Adding ros_yesense_imu to documentation index for kinetic

### PR DESCRIPTION
**Pull request by wangzhecoder**
I'd like my ros_yesense_imu to be indexed and documented on ros.org

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -31,11 +31,11 @@ repositories:
       - abb_resources
       tags:
         release: release/kinetic/{package}/{version}
-      url: https://github.com/ros-industrial-release/abb-release.git
+      url: https://github.com/ros-industrial/abb.git
```
Comment (dirk-thomas) [inline kinetic/distribution.yaml]: Please revert this unrelated change.

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -31,11 +31,11 @@ repositories:
...
       version: 1.3.0-1
     source:
       type: git
-      url: https://github.com/ros-industrial/abb.git
+      url: https://github.com/wangzhecoder/ros_yesense_imu.git
```
Comment (dirk-thomas) [inline kinetic/distribution.yaml]: Same here.

**Review by dirk-thomas — CHANGES_REQUESTED** [review]
See other comments.

**Comment by tfoote** [comment]
Closing due to no response in over a week. Please comment back here and we can reopen if you remove the unrelated changes from the PR. We'd be happy to index your project but your PR is mutating other packages.

**Pull request closed without merging**